gtsfm/loader/one_d_sfm_loader.py

```diff
@@ -101,9 +129,14 @@ def get_camera_intrinsics_full_res(self, index: int) -> Optional[Cal3Bundler]:
             Intrinsics for the given camera.
         """
         # Get intrinsics from exif.
-        intrinsics = io_utils.load_image(self._image_paths[index]).get_intrinsics_from_exif(
-            default_focal_length_factor=NO_EXIF_DEFAULT_FOCAL_LENGTH_FACTOR
-        )
+        # TODO(yanwei) It is probably fine to only use the second method, as
+        # the images have been filtered before in consturctor.
+        if not self._enable_no_exif:
+            intrinsics = io_utils.load_image(self._image_paths[index]).get_intrinsics_from_exif()
+        else:
+            intrinsics = io_utils.load_image(self._image_paths[index]).get_intrinsics(
+                default_focal_length_factor=NO_EXIF_DEFAULT_FOCAL_LENGTH_FACTOR
```

Comment:
I agree with your TODO that I don't think its necessary to check the exif flag again

Reply:
Done.